packages/node_modules/@node-red/nodes/core/function/10-function.js

```diff
@@ -166,7 +169,15 @@ module.exports = function(RED) {
             Buffer:Buffer,
             Date: Date,
             RED: {
-                util: RED.util
+                util: {
+                    ...RED.util,
+                    getSetting: function (node, name, flow) {
+                        if (envVarExcludes[name]) {
+                            return undefined;
+                        }
+                        return RED.util.getSetting(node, name, flow);
+                    }
```

Comment:
```suggestion
                    getSetting: function (_node, name, _flow) {
                        // Ensure `node` argument is the Function node and do not allow flow to be overridden.
                        return RED.util.getSetting(node, name);
                    }
```

Reply:
I should have done it like this from the start...